pyccel/parser/base.py

```diff
@@ -201,15 +202,10 @@ def loops(self):
 
 class BasicParser(object):
 
-    """ Class for a base Parser."""
+    """ Class for a base Parser.
+    This class contains functions and properties which are common to SyntacticParser and SemanticParser
 
-    def __init__(self,
-                 debug=False,
-                 headers=None,
-                 static=None,
-                 show_traceback=False,
-                 output_folder=''):
-        """Parser constructor.
+    Parser constructor:
```

Comment:
Here you should just write
```
Parameters
----------
```
see https://numpydoc.readthedocs.io/en/latest/format.html#class-docstring.